### PR TITLE
fix: Support http2, grpc protocols in canary release request content …

### PR DESCRIPTION
**Pull request by leoendless**
…policy

Signed-off-by: leoendless <leoliu@yunify.com>

**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by zackzhangkai** [comment]
https://istio.io/latest/docs/concepts/traffic-management/#routing-rules

**Comment by imjoey** [comment]
Much appreciated to @zackzhangkai for kind help, as well as this great PR for @leoendless .

File: locales/es/grayrelease.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ module.exports = {
   'Please select a grayscale release component':
     'Selecciona un componente Grayscale Release',
   'Policy Config': 'Configuración de política',
-  'Port protocol is not HTTP, cannot publish policy by request content':
+  POLICY_REQUEST_CONTENT_TIP:
     'El protocolo de puerto no es HTTP, no se puede publicar la política por el contenido de la solicitud',
```
Comment (imjoey) [inline locales/es/grayrelease.js:62]: @leoendless Hi, I'm afraid that we also need to add the `HTTP2` and `gRPC` tips here in Spanish locales.

**Pull request merged**